build.cake

```diff
@@ -137,15 +145,26 @@ Task("Copy-Files")
     .IsDependentOn("Run-Unit-Tests")
     .Does(() =>
 	{
-		EnsureDirectoryExists(parameters.ResultBinDir);
-
-		CopyFileToDirectory(parameters.BuildDir + "/Cake.SqlServer.dll", parameters.ResultBinDir);
-		CopyFileToDirectory(parameters.BuildDir + "/Cake.SqlServer.pdb", parameters.ResultBinDir);
-		CopyFileToDirectory(parameters.BuildDir + "/Cake.SqlServer.xml", parameters.ResultBinDir);
-
-		CopyFiles(parameters.BuildDir + "/Microsoft.*.dll", parameters.ResultBinDir);

```

Comment:
I'm pretty sure we still need all the `Microsoft.*.dll` files for bacpac operations - that was a major issue in the past, I don't think much have changed since last time I looked on this.

Reply:
thanks for heads up - I will check this - I wasn't aware